server/views/partials/nav.pug

```diff
@@ -0,0 +1,23 @@
+// Navigation
+nav#mainNav.navbar.navbar-expand-lg.bg-secondary.fixed-top.text-uppercase
+  .container
+    a.navbar-brand.js-scroll-trigger(href='#page-top') Start Bootstrap
+    button.navbar-toggler.navbar-toggler-right.text-uppercase.bg-primary.text-white.rounded(type='button', data-toggle='collapse', data-target='#navbarResponsive', aria-controls='navbarResponsive', aria-expanded='false', aria-label='Toggle navigation')
+      | Menu
+      i.fas.fa-bars
+    #navbarResponsive.collapse.navbar-collapse
+      ul.navbar-nav.ml-auto
+        if (!session || !session.userId)
```

Comment:
Maybe we could just use something related to `req.user`? `req.user` only exists if passport has authenticated them.